src/server/gql/schema/index.ts

```diff
@@ -0,0 +1,81 @@
+import { IResolvers, makeExecutableSchema } from 'graphql-tools'
+import { applyMiddleware } from 'graphql-middleware'
+import merge from 'lodash.merge'
+import { shield } from 'graphql-shield'
+import { gql } from 'apollo-server-express'
+import * as GqlScalars from 'graphql-scalars'
+import * as User from './User'
+import * as Email from './Email'
+import { GQL_SHIELD_DEBUG } from '../../environment'
+import prisma from '../prisma'
+import { isSelf } from '../shield'
+
+const typeDefs = gql`
+  type Query {
+    email(email: String!): Email
+  }
+  type Mutation {
+    addUser(email: String!, givenName: String!, familyName: String!): User
+    claimSystemAdmin(email: String!): User
+  }
+`
+
+const resolvers: IResolvers = {
+  Mutation: {
+    claimSystemAdmin: async (parent, { email }: { email: string }) =>
+      prisma.$transaction(async (txPrisma) => {
+        const assignments = await txPrisma.systemRoleAssignment.count({
+          where: {
+            role: { name: 'admin' },
+          },
+        })
+        if (assignments > 0) {
+          throw new Error('someone else has already claimed systemAdmin')
```

Comment:
**Question**

Why only allow one person to be the admin?

Reply:
I was testing. I'm actually going to rove this from now.